### PR TITLE
ON-32841 # Added ListsClient and LookupsClient

### PR DESCRIPTION
**Pull request by jdawg093**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ SECRET_KEY=YOUR_SECRET_KEY
 
 ### Target frameworks
 
-By default the tests will run agaisnt runtimes .Net Core 3.1, .Net 5.0, and .Net 6.0
```
Comment (kizaonline) [inline CONTRIBUTING.md:34]: Since your changing this, the versions listed is out of date, its should be `.Net 6.0 and .Net 7.0`

Comment (jdawg093) [reply CONTRIBUTING.md:34]: Did you want this changed in all places of the file?

Comment (kizaonline) [reply CONTRIBUTING.md:34]: Yes, every place it is applicable

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ SECRET_KEY=YOUR_SECRET_KEY
 
 ### Target frameworks
 
-By default the tests will run against runtimes .Net Core 3.1, .Net 5.0, and .Net 6.0
+By default the tests will run against runtimes .Net 6.0, and .Net 8.0
```
Comment (kizaonline) [inline CONTRIBUTING.md:34]:
```suggestion
By default the tests will run against runtimes .Net 6.0, and .Net 7.0
```

File: OneBlink.SDK/models/FormElementList.cs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        {
+            get; set;
+        }
+    }
+
+    //This is the result we return from the `Search()` function in ListsClient.
+    //This is to avoid mention to the older name of `OptionSets` for `Lists`.
+    public class FormElementListSearchResult : SearchResult
+    {
+        public List<FormElementList> FormElementLists
```
Comment (Zaxist) [inline OneBlink.SDK/models/FormElementList.cs:57]: Should be lower case starting letter, no? 
```
formElementLists
```

Comment (jdawg093) [reply OneBlink.SDK/models/FormElementList.cs:57]: You would in fact be correct, I'll fix this post haste

**Pull request merged**